client/src/App.js

```diff
@@ -21,8 +21,14 @@ export default class App extends Component {
   render() {
     return (
       <div>
+
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path='/' component={MovieList}/>
+        <Route
+          path='/:movieId'
+          render={(props) => <Movie {...props}
+            addToSavedList={this.addToSavedList} />}
```

Comment:
Right now `addToSavedList` adds a movie even though that movie is already on the `savedList`, for a challenge, can you modify the function to only add a movie if it is not in the `savedList`.